swupd/manifest.go

```diff
@@ -542,7 +542,9 @@ func deprecateOldRenames(oldRenames []*File) {
 	for _, f := range oldRenames {
 		if f.DeltaPeer == nil {
 			f.Rename = false
-			f.Type = TypeUnset
+			if !f.Present() {
```

Comment:
Told you we should not set the type to TypeUnset.

Reply:
yeah yeah...

Reply:
😭